run-e2e-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+go test ../tests/e2e/ -timeout 0
```

Comment:
Should we add args into  `make test-ccm-e2e` instead? Is make working in the same way?

Reply:
but with `make test-ccm-e2e` you cant do `exec.Command("./hack/run-e2e-tests.sh", testArgs...)` where it's expecting a string and a string slice. https://golang.org/pkg/os/exec/#Command I was hoping we can reuse `testArgs` from test-infra.

Reply:
The issue is when we use `make`, we have to trigger the call like `exec.Command("make", "test-ccm-e2e")` which won't work for `testArgs...`.  To use `testArgs`, we have to pass in a single string to `exec.Command` first, then followed by `testArgs...)` like `exec.Command("./hack/run-e2e-tests.sh", testArgs...)`